bin/crew

```diff
@@ -334,6 +380,8 @@ def help (pkgName)
     puts "Update crew."
     puts "Usage: crew update"
     puts "This only updates crew itself.  Use `crew upgrade` to update packages."
+    puts "Usage: crew update compatible"
+    puts "This updates the crew package compatibility list."
```

Comment:
Why do we need to do this manually?  Shouldn't it be part of the `crew update` process?

Reply:
It still is part of that process. This just allows one to run it separately without a git pull, which is incredibly useful when you are making local changes to files on a git repo and want to see if there are going to be issues with the package files being read correctly.

If we had had this before, we would have avoided many issues with crew breaking. :)